src/twisted/web/distrib.py

```diff
@@ -62,17 +63,17 @@ def setCopyableState(self, state):
         state['requestHeaders'] = Headers(dict(state['requestHeaders']))
         pb.RemoteCopy.setCopyableState(self, state)
         # Emulate the local request interface --
-        self.content = cStringIO.StringIO(self.content_data)
-        self.finish           = self.remote.remoteMethod('finish')
-        self.setHeader        = self.remote.remoteMethod('setHeader')
-        self.addCookie        = self.remote.remoteMethod('addCookie')
-        self.setETag          = self.remote.remoteMethod('setETag')
-        self.setResponseCode  = self.remote.remoteMethod('setResponseCode')
-        self.setLastModified  = self.remote.remoteMethod('setLastModified')
+        self.content = BytesIO(self.content_data)
+        self.finish           = self.remote.remoteMethod(b'finish')
```

Comment:
`RemoteMethod` really ought to take a `str`.  A better place to fix this would be `RemoteMethod`'s constructor.